lib/model/migrations/20180727-02-add-md5-to-blobs.js

```diff
@@ -11,7 +11,7 @@
 const { md5sum } = require('../../util/crypto');
 
 const up = (knex) =>
-  knex.schema.table('blobs', (blobs) => { blobs.string('md5', 32); })
+  knex.schema.table('blobs', (blobs) => { blobs.string('md5', 32).notNull(); })
```

Comment:
I think this migration may fail if the `blobs` table is not empty: I think `knex` will apply the `not null` constraint before running the backfill. Maybe the `not null` constraint could go in a separate migration so that it's applied only after the backfill? Alternatively, maybe we could set a column default of `''` rather than `null` so that rows don't initially violate the constraint.